Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -107,6 +110,19 @@ TEST_CASE(test_bmp)
     TRY_OR_FAIL((test_roundtrip<Gfx::BMPWriter, Gfx::BMPImageDecoderPlugin>(TRY_OR_FAIL(create_test_rgba_bitmap()))));
 }
 
+TEST_CASE(test_gif)
+{
+    // We only support grayscale and non-animated images yet
+    auto bitmap = TRY_OR_FAIL(create_test_rgb_bitmap());
+
+    // Convert bitmap to grayscale
+    for (auto& argb : *bitmap)
+        argb = Color::from_argb(argb).to_grayscale().value();
+
+    bool const is_gif = true;
```

Comment:
I get the "no magic constants" guideline, but this feels a bit excessive. (And in case you want to keep it, this should probably be `constexpr`.)

Reply:
I removed it.